spec/features/variables/empty_response_when_setting_variable_spec.rb

```diff
@@ -41,26 +39,15 @@
     end
 
     context 'when visiting the show page of a non-existent variable' do
-      # This test should be revisited with MMT-2301. MMT-1557 made a change that if a concept cannot be retrieved to compare UMM Versions, editing is not allowed so we are only showing a page notifying the user the version cannot be confirmed
       before do
-        visit variable_path('fake-concept-id')
+        visit variable_path('Vfake-concept-id')
```

Comment:
I think if this `Vfake-concept-id` was changed to `var-fake-concept-id` and the `Sfake-concept-id` was changed to `service-fake-concept-id` it may increase readability, because at first I wasn't sure if it was a typo or intentional. A comment may help too

Reply:
I will add comments.  They need to contain the 'signature' letter (V or S) for the CMR concept to be parsed correctly in set_metadata.